From e9c8bb3f2bf792fd63f3ab2e1f5333267d1155fe Mon Sep 17 00:00:00 2001
From: Hans-Christian Egtvedt <hans-christian.egtvedt@atmel.com>
Date: Wed, 3 Nov 2010 10:25:45 +0100
Subject: [PATCH 1/1] avr32: use generic syscall defines instead of arch specific

This patch removes the duplicated syscall function defines in syscalls.h and
relies on the generic defines in the asm-generic directory. This fix makes
AVR32 architecture compile again, as the duplicated syscalls.h functions came
in conflict with other patches.

Signed-off-by: Hans-Christian Egtvedt <hans-christian.egtvedt@atmel.com>
---
 arch/avr32/include/asm/syscalls.h |    9 ---------
 1 files changed, 0 insertions(+), 9 deletions(-)

--- a/arch/avr32/include/asm/syscalls.h
+++ b/arch/avr32/include/asm/syscalls.h
@@ -16,18 +16,9 @@
 #include <linux/signal.h>
 
 /* kernel/process.c */
-asmlinkage int sys_fork(struct pt_regs *);
 asmlinkage int sys_clone(unsigned long, unsigned long,
 			 unsigned long, unsigned long,
 			 struct pt_regs *);
-asmlinkage int sys_vfork(struct pt_regs *);
-asmlinkage int sys_execve(const char __user *, char __user *__user *,
-			  char __user *__user *, struct pt_regs *);
-
-/* kernel/signal.c */
-asmlinkage int sys_sigaltstack(const stack_t __user *, stack_t __user *,
-			       struct pt_regs *);
-asmlinkage int sys_rt_sigreturn(struct pt_regs *);
 
 /* mm/cache.c */
 asmlinkage int sys_cacheflush(int, void __user *, size_t);
